include/rcutils/error_handling.h

```diff
@@ -314,6 +314,93 @@ RCUTILS_PUBLIC
 void
 rcutils_reset_error(void);
 
+// Helper macro for nested macro expansion
+#define _RCUTILS_EXPAND(x) x
```

Comment:
I'm not sure, but maybe this is what you want?

https://github.com/ros2/rcutils/blob/5bd1230eb10595cec30168667adf03f48879b96c/include/rcutils/macros.h#L68

Reply:
It's not unfortunately :(, I was having a case where something like `MACRO(INNER_MACRO(msg ": %s"))` was becoming `MACRO(msg ": %s")` instead of expanding the `msg` arg.

Stringify almost does that, but causes the expansion to become `"<expanded_msg> ": %s""` (note the double-double quotes)

Reply:
I'll move this to `macros.h`, though